src/org/traccar/database/ConnectionManager.java

```diff
@@ -41,6 +41,7 @@
 
     private final long deviceTimeout;
     private final boolean enableStatusEvents;
+    private final boolean correctMotionState;
```

Comment:
Can't think of any good name at the moment, we definitely need something better here.